manim/mobject/mobject.py

```diff
@@ -1721,33 +1724,39 @@ def length_over_dim(self, dim):
         ) - self.reduce_across_dimension(np.min, np.min, dim)
 
     def get_coord(self, dim, direction=ORIGIN):
-        """Meant to generalize get_x, get_y, get_z"""
+        """Meant to generalize ``get_x``, ``get_y`` and ``get_z``"""
         return self.get_extremum_along_dim(dim=dim, key=direction[dim])
 
     def get_x(self, direction=ORIGIN):
+        """Returns x coordinate of the center of the ``mobject`` as ``numpy.float64`` """
         return self.get_coord(0, direction)
 
     def get_y(self, direction=ORIGIN):
+        """Returns y coordinate of the center of the ``mobject`` as ``numpy.float64`` """
         return self.get_coord(1, direction)
 
     def get_z(self, direction=ORIGIN):
+        """Returns z coordinate of the center of the ``mobject`` as ``numpy.float64`` """
```

Comment:
should this be shorter?
At the moment, I think it looks a bit to long: 
![image](https://user-images.githubusercontent.com/44469195/113142819-a1290080-922b-11eb-8c99-ad57fcca4231.png)

Reply:
> 
> 
> https://github.com/ManimCommunity/manim/blob/c3f6119761a10991d45cbd46b218c9dc54d037e1/docs/source/manim_directive.py#L135
> 
> Found it!
> ok, I can add this to this pr.

I just made #1208, feel free to do it here if you wish, but I'd still suggest a separate PR if this one is supposed to be for documentation/docstrings